lib/collection/src/collection_manager/optimizers/segment_optimizer.rs

```diff
@@ -38,6 +39,18 @@ pub struct OptimizerThresholds {
     pub indexing_threshold_kb: usize,
 }
 
+fn available_disk_space(path: &Path) -> u64 {
+    let mut sys = System::new_all();
+    sys.refresh_disks_list();
+
+    for disk in sys.disks() {
```

Comment:
We are already using `fs4` for he similar purpose https://github.com/qdrant/qdrant/blob/1efec29f65c7569a3ae3b76bbd511410556c53c6/lib/collection/src/shards/local_shard/disk_usage_watcher.rs#L106

I would prefer to keep the consistency